x-pack/plugins/search_playground/public/components/app.tsx

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { KibanaPageTemplate } from '@kbn/shared-ux-page-kibana-template';
+
+import { Chat } from './chat';
+
+export const SearchPlaygroundApp: React.FC = () => {
```

Comment:
nitpik: rename the file to `search_playground_app`?

Reply:
Changed component name to App it makes more sense